symphony/bdk/core/config/model/bdk_datahose_config.py

```diff
@@ -0,0 +1,24 @@
+from symphony.bdk.core.config.model.bdk_retry_config import BdkRetryConfig
+
+TAG = "tag"
+FILTERS = "filters"
+RETRY = "retry"
+
+
+class BdkDatahoseConfig:
+    """Class holding datahose specific configuration.
+    """
+
+    def __init__(self, config):
+        """
+
+        :param config: the dict containing the datahose specific configuration.
+        """
+        self.tag = None
+        self.filters = None
+        self.retry = BdkRetryConfig(dict(maxAttempts=BdkRetryConfig.INFINITE_MAX_ATTEMPTS))
+        if config is not None:
+            self.tag = config.get(TAG)
```

Comment:
We should set a default value for tag and filters if config is None (maybe None value).

Reply:
Isn't it the case here? we first initialise tag and filters with None and "retry" with default one, and then we override them if config is not None.
The default tag (datahose-BOT_NAME) or (datahose) is set in DatahoseLoop constuctor 

https://github.com/finos/symphony-bdk-python/pull/281/files/33baa811ecb26d83f22300c833b94e43e46ae4a3#diff-c036be8e109b086eabf605d69cf2d381e5375847f1d086bb4942784f529fe247R36